x-pack/test/functional/apps/visualize/reporting.ts

```diff
@@ -138,7 +138,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
         await PageObjects.reporting.clickGenerateReportButton();
 
         log.debug('get the report download URL');
-        const url = await PageObjects.reporting.getReportURL(60000);
+        const url = await PageObjects.reporting.getReportURL(120000);
```

Comment:
why do we have to increase the timeout here?

Reply:
The new embeddable system seems to take longer to generate a screenshot in the CI. Increased this timeout to eliminate flakiness.

Reply:
Apparently this test is now skipped on `main` for unrelated issues: https://github.com/elastic/kibana/issues/189590

I could revert this change and this PR would still technically pass CI, but I'm pretty sure that would reintroduce flakiness problems as soon as we try to fix #189590